tests/test_mag_orms.py

```diff
@@ -10,7 +10,7 @@
 
 class TestMag(unittest.TestCase):
     '''Check that the MAG ORM works as expected'''
-    engine = create_engine(os.getenv("postgresdb"))
+    engine = create_engine(os.getenv("test_postgresdb"))
     Session = sessionmaker(engine)
 
     def setUp(self):
```

Comment:
Picky thing: Class names should be CamelCase but method names should be snake_case

Reply:
Ok, now I see that this is the fault of `unittest`.